pkg/webhook/cluster/mutating_webhook.go

```diff
@@ -217,10 +217,10 @@ func newTaintTimeAddedJsonPatch(index int, timeAdded time.Time) jsonPatchOperati
 	}
 }
 
-func newLabelJsonPatch() jsonPatchOperation {
+func newLabelJsonPatch(labels map[string]string) jsonPatchOperation {
 	return jsonPatchOperation{
-		Operation: "add",
+		Operation: "replace",
```

Comment:
If the original labels map is nil, will this work?

Reply:
it could not be nil,
As this must be has `clusterSetLabels[clusterSetLabel] = defaultClusterSetName`

Reply:
I mean the **original** object, this func just returns a json patch, right？ where will the json patch be used? Could this patch apply to the ManagedCluster object(labels nil) successfully？